src/betterproto/plugin/models.py

```diff
@@ -503,16 +504,13 @@ def __post_init__(self):
                     self.proto_v_type = self.proto_obj.Type.Name(nested.field[1].type)
         super().__post_init__()  # call FieldCompiler-> MessageCompiler __post_init__
 
-    def get_field_string(self, indent: int = 4) -> str:
```

Comment:
Could you explain a bit what this change is about? I'm not seeing the full picture here.

Reply:
I realized that we could easily re-use the `get_field_string` machinery in the base class instead of overwriting it.